client/Makefile

```diff
@@ -3,22 +3,24 @@ SUBDIRS = doc po
 PREFIX = /usr
 BINDIR = $(PREFIX)/bin
 LIBDIR = $(PREFIX)/lib
+PYTHON_SITEDIR = $(LIBDIR)
 DESTDIR = 
 
 all:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d; done
 
 install:
-	install -d $(DESTDIR)/$(LIBDIR)/entropy/client/solo/commands
+	install -d $(DESTDIR)/$(LIBDIR)/entropy/client
+	install -d $(DESTDIR)/$(PYTHON_SITEDIR)/entropy/client/solo/commands
 	install -d -m 775 $(DESTDIR)/etc/entropy
 	install -d $(DESTDIR)$(BINDIR)
 	install -d $(DESTDIR)$(PREFIX)/share/man/man1
 
 	# copying portage bashrc
 	install -m 644 ../conf/client.conf $(DESTDIR)/etc/entropy/
 
-	install -m 644 solo/*.py $(DESTDIR)/$(LIBDIR)/entropy/client/solo/
-	install -m 644 solo/commands/*.py $(DESTDIR)/$(LIBDIR)/entropy/client/solo/commands/
+	install -m 644 solo/*.py $(DESTDIR)/$(PYTHON_SITEDIR)/entropy/client/solo/
+	install -m 644 solo/commands/*.py $(DESTDIR)/$(PYTHON_SITEDIR)/entropy/client/solo/commands/
 	install -m 644 revision $(DESTDIR)/$(LIBDIR)/entropy/client/
```

Comment:
shouldn't also this go inside ```$(PYTHON_SITEDIR)```?

Reply:
No, it's a file with version and has to be in a well known location Entropy can find. (It's not a Python file.)